### ISSUE TITLE
Keys represented in photon_layouts

### ISSUE THREAD

**Issue by sypl** [jacekplonowski/TAS_kc#3]
I'm wondering which keys are in each file in the photon layouts folder: https://github.com/jacekplonowski/TAS_kc/tree/master/layouts/SLA/photon_layouts

Each file has nine keys, but they're not labelled anywhere. I'm not sure if I'm meant to print out all three for a set, but that doesn't make sense as that gives me 27 keys. I also notice b set has a 1.5u key, so I'm guessing it's a variation of a set?

**Comment by jacekplonowski** [comment]
Hello,

Pardon me for the confusion, I do agree the premade STL layouts are insufficiently documented for practical usage.

Unfortunately, I am no longer involved in keyboard/keycap design and the respective communities, so this repository is more of an archive of the files rather than an active project.

However, I can try to address some of the questions here from what I can remember when I was still actively working on this project. The three layouts (a, b, and c) are how I choose to split up the printing into batches. To fill a Corne-like 4x6 split keyboard, I printed two copies of each layout (e.g. 2x a, 2x b, 2x c) which left me with enough keys to fill a "4x12" board plus some spares. I don't think I'll be able to tell you exactly which keys are contained in each layout as it has been a long time since I created those layouts, but I essentially tried to split up half of the 4x6 split keyboard  into 3 batches of 9 keys, which I found was the upper limit for my printer at the time.

Going forward, I could try to help you tailor a set specific to your keyboard, which would be in composed of the core keys (the 3x6 grid as shown in the projects readme) plus some other keys to act as the modifiers, the spacebar, and layer keys. If that sounds like something you're interested in, let me know what sort of keyboard you have and I could try to come up with a suggestion of which keycap models to print, but arranging them into printable layouts would be left to you.

**Comment by sypl** [comment]
Thanks for the prompt reply. I think I'll try and make my own print sets (for a planck). You gave me advice on reddit a while back (your user is deleted now?) on creating supports in Prusa, which I think I have the hang of, so I'll try that. I'll come back here if I can't get it working.

**Comment by jacekplonowski** [comment]
Yes, I was just inactive on the site and decided to close down the account. Best of luck on creating the layouts!
